src/tui/rendering.rs

```diff
@@ -37,12 +37,13 @@ impl Theme {
 }
 
 pub struct Tui {
-    process_table: TableState,
-    process_table_scroll: ScrollbarState,
     theme: Theme,
-    number_of_items: usize,
-    details_scroll_state: ScrollbarState,
-    process_details_scroll: u16,
+    process_table: TableState,
+    process_table_scroll_state: ScrollbarState,
+    process_table_number_of_items: usize,
+    process_details_scroll_state: ScrollbarState,
+    process_details_scroll_offset: u16,
+    process_details_number_of_lines: u16,
```

Comment:
I slightly rearranged and renamed these variables to make naming more uniform. I understand if you want to push these changes back to keep the old naming.

Reply:
It's ok